### PR TITLE
Add a new file about hometown and delete the old useless one

### LINKED ISSUE #5: Motun tracks progress on GitHub flow
- [x] create a **branch** (git)
- [x] add some **commits** (git)
- [x] open a *pull request* (GitHub)
- [x] additional **commits** (git)
- [ ] _merge_ (git)

### PR DESCRIPTION
**Pull request by Motun**
This PR adds an Ankara file to the class repo. This PR closes #5

### CODE REVIEW

File: Motun-Ankara.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Ankara, Turkey
```
Comment (nvrdftd) [inline Motun-Ankara.md]: @Motun Is Ankara the capital city of your country? I hope someday I will travel to Istanbul.

Comment (Motun) [inline Motun-Ankara.md]: Yes, it is. I strongly suggest you visit Istanbul, one of the most beautiful cities in the world!

**Comment by Motun** [comment]
I'll merge when I get a :+1: , :anchor: or :100:

**Comment by neonblueflame** [comment]
:+1:

**Pull request merged**